pkg/codegen/internal/test/testdata/simple-resource-schema/nodejs/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@pulumi/example",
-    "version": "${VERSION}",
+    "version": "1.2.3",
```

Comment:
There were a number of attempts in the past to set this during codegen but every time we concluded we needed to do that in CI to avoid spurious diffs (e.g. when each PR bumps to a git-hash-based version, new every time). Has this changed?

Reply:
This has not changed. That is why the feature is off by default. The feature is targeted at external (non-pulumi) customers of codegen.

Reply:
Thank you, that makes sense.